pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -153,11 +153,11 @@ type Jaeger struct {
 // +k8s:openapi-gen=true
 type JaegerCommonSpec struct {
 	// +optional
-	// +listType=set
+	// +listType=atomic
```

Comment:
Why these changes?

Reply:
There was a change in how OpenAPI and the generated resources would validate the values. I don't remember exactly the reason, as this is something I faced during the upgrade of the OpenTelemetry Operator as well: https://github.com/open-telemetry/opentelemetry-operator/pull/24/files#diff-0f1140984003d7bcbaddb3717895a48d

Reply:
ok thanks